src/Cronofy/Conferencing.cs

```diff
@@ -0,0 +1,78 @@
+namespace Cronofy
+{
+    /// <summary>
+    /// Class for the deserialization of the options for a read event
+    /// response.
+    /// </summary>
+    public sealed class Conferencing
+    {
+        /// <summary>
+        /// Gets or sets the conferencing provider name.
+        /// </summary>
+        /// <value>
+        /// The conferencing provider name.
+        /// </value>
+        public string ProviderName { get; set; }
```

Comment:
Like what has been done for `AttendeeStatus` we could create a static class of ConferencingProviders containing the possible string values of providers, and reference that class in the docxml.